Sources/ProjectDescription/Cloud.swift

```diff
@@ -4,14 +4,6 @@ import Foundation
 public struct Cloud: Codable, Equatable {
     /// Options for cloud configuration.
     public enum Option: String, Codable, Equatable {
-        /// Enables sending analytics to cloud dashboard.
-        @available(
-            *,
-            deprecated,
-            message: "Analytics are sent to the cloud backend by default. Use `disableAnalytics` to disable this feature."
-        )
-        case analytics
-
         /// Disables sending analytics to cloud dashboard.
         case disableAnalytics
```

Comment:
While you're at it, can you remove this case as well? Tuist Cloud won't work properly if you disable analytics (as they are a core part of the functionality).